portugues/ocupado.md

```diff
@@ -0,0 +1,5 @@
+Só dá ocupado.
+
+[Você pega um ônibus até o centro para comprar a ultima versão XTree, mas você só tem Cr$ 5,00 na carteira.](centro.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.